global/prometheus-alertmanager-operated/templates/am-config-route-slack.yaml

```
@@ -467,6 +467,26 @@ spec:
             matchType: "="
             value: compute
 
+      # Test Channel for CC KVM Alerting slack_alert_kvm_test
+      - receiver: slack_alert_kvm_test
+        continue: true
+        matchers:
+          - name: tier
+            matchType: "="
+            value: vmware
+          - name: support_group
+            matchType: "="
+            value: compute
+          - name: service
+            matchType: "="
+            value: kvm
+          - name: severity
+            matchType: "="
```

Regex matches require a different `matchType`.
```suggestion
            matchType: "=~"
```